contact.html

```diff
@@ -146,10 +146,10 @@ <h2 class="subtitle">Get in touch</h2>
                 <input type="checkbox" class="input-hidden form-checkbox__input" required>
                 <p class="form-checkbox__inner">
                 <span class="form-checkbox__box"></span>
-                <span class="form-checkbox__text">By clicking on the ‘Send Message’ button I agree to <a target="_blank">all the terms of the security policy</a></span>
```

Comment:
Please move margin 5px from :checked and :invalid state to default checkbox state

<img width="1317" alt="Screenshot 2024-12-16 at 01 41 57" src="https://github.com/user-attachments/assets/178d86ea-dd9b-487c-bb68-69c4b10b862d" />

Reply:
done